tailwind.config.js

```diff
@@ -2,6 +2,23 @@ const defaultTheme = require('tailwindcss/defaultTheme')
 
 module.exports = {
   theme: {
+    colors: {
+      white: '#ffffff',
+      green: {
+        primary: '#18bc9c',
+        dark: '#15a589',
+        darker: '#12826c'
+      },
+      gray: {
+        lightest: '#fafbfc',
+        lighter: '#f1f3f5',
+        light: '#eef3f6',
+        primary: '#d7dee0',
+        dark: '#636f7b',
+        darker: '#30373f',
+        darkest: '#1c2024'
```

Comment:
Hmm, maybe we should just use the Tailwind defaults instead of these grays? The greens are perfectly ok.

Reply:
I was just trying to match  the colours in the article, but if you'd rather stick to defaults, I'm happy to do that.

Reply:
Yeah I think so. I mean if there's not too much difference that is.